src/js_parser.zig

```diff
@@ -3100,6 +3144,8 @@ pub const Parser = struct {
 
         transform_only: bool = false,
 
+        target: options.Target = .bun,
```

Comment:
What is this doing? We have been going feature (supports runtime require) rather than target (bun) . Should this be in Runtime.Features?